.gitignore

```diff
@@ -106,3 +106,6 @@ venv.bak/
 # project specific
 /projects
 /output
+
+#Pycharm
+.idea
```

Comment:
You can do this in your global gitignore to avoid having to do this in each project :)

Reply:
Didn't know that actually